test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -4260,5 +4260,129 @@ private void ValidatePackIcon(ProjectFileBuilder projectBuilder)
                 Assert.Equal(projectBuilder.PackageIcon, nuspecReader.GetIcon());
             }
         }
+
+        [PlatformTheory(Platform.Windows)]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void PackCommand_RequireLicenseAcceptanceNotEmittedWhenUnspecified(bool withLicense)
```

Comment:
Merge the tests in one method. Creating a new project from those tests is somewhat time consuming. It is better to modify the project file when needed and make the required assertions.